core/block/export/export.go

```diff
@@ -274,12 +274,25 @@ func (e *export) getExistedObjects(includeArchived bool, isProtobuf bool) (map[s
 	return objectDetails, nil
 }
 
-func (e *export) writeMultiDoc(mw converter.MultiConverter, wr writer, docs map[string]*types.Struct, queue process.Queue) (succeed int, err error) {
+func (e *export) writeMultiDoc(mw converter.MultiConverter, wr writer, docs map[string]*types.Struct, queue process.Queue, includeFiles bool) (succeed int, err error) {
 	for did := range docs {
 		if err = queue.Wait(func() {
 			log.With("objectID", did).Debugf("write doc")
 			werr := e.bs.Do(did, func(b sb.SmartBlock) error {
-				return mw.Add(b.NewState().Copy())
+				if err = mw.Add(b.NewState().Copy()); err != nil {
+					return err
+				}
+				if !includeFiles {
+					return nil
+				}
+				for _, fh := range b.GetAndUnsetFileKeys() {
+					if werr := e.saveFile(wr, fh.Hash); werr != nil {
+						if werr = e.saveImage(wr, fh.Hash); werr != nil {
```

Comment:
If we got error while saving file it's not implying that we have an image. Maybe just call safe file? Or collect image hashes separately, but I dont' think it's necessary